app/src/main/java/com/it3048/mobiledeviceproject/MainActivity.kt

```diff
@@ -19,7 +19,7 @@ import java.util.*
 
 
 class MainActivity : AppCompatActivity() {
-    var addMeetingForm = AddMeetingForm()
+    private var addMeetingForm = AddMeetingForm()
```

Comment:
Updated visibility modifiers per the suggestions here: https://kotlinlang.org/docs/visibility-modifiers.html#packages